packages/base/src/widget.ts

```diff
@@ -450,11 +450,12 @@ class WidgetModel extends Backbone.Model {
      *
      * This invokes a Backbone.Sync.
      */
-    save_changes(callbacks?) {
```

Comment:
The callbacks here are Jupyter callbacks, and don't have to do with Backbone. See where we use them in https://github.com/jupyter-widgets/ipywidgets/blob/master/packages/base/src/widget.ts#L358